packages/typedoc-plugins/__tests__/interface-augmentation-fixer/fixtures/foobar/foobarconfig.ts

```
@@ -0,0 +1,27 @@
+        propertyFoobarSource: number;
+    }
+}
+
+// Module augmentation that points to the re-exported interface in "index.ts" file.
+declare module "../" {
+    interface FooConfig {
+        propertyFoobarIndex: number;
+    }
+}
+
+export interface FoobarConfig {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)